dev-docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
I think we can probably nuke this file? Most of our tooling assumes a *nix system.